projects/opendr_ws/src/perception/scripts/skeleton_based_action_recognition.py

```diff
@@ -188,16 +191,46 @@ def _pose2numpy(num_current_frames, poses_list):
 
 
 if __name__ == '__main__':
+
+    parser = argparse.ArgumentParser()
+    parser.add_argument("-i", "--input_rgb_image_topic", help="Topic name for input image",
+                        type=str, default="/usb_cam/image_raw")
+    parser.add_argument("-o", "--output_rgb_image_topic", help="Topic name for output annotated image",
+                        type=str, default="/opendr/image_pose_annotated")
+    parser.add_argument("-p", "--pose_annotations_topic", help="Topic name for pose annotations",
+                        type=str, default="/opendr/poses")
+    parser.add_argument("-c", "--output_category_topic", help="Topic name for recognized action category",
+                        type=str, default="/opendr/skeleton_recognized_action")
+    parser.add_argument("-d", "--output_category_description_topic", help="Topic name for description of the "
+                                                                          "recognized action category",
+                        type=str, default="/opendr/skeleton_recognized_action_description")
```

Comment:
This is a new addition in #306, which i didn't mention in the tracker issue (will edit it in). It serves to handle the `None` values passed through argparse as string type instead of none type.

```suggestion
    parser.add_argument("-o", "--output_rgb_image_topic", help="Topic name for output annotated image",
                        type=lambda value: value if value.lower() != "none" else None, 
                        default="/opendr/image_pose_annotated")
    parser.add_argument("-p", "--pose_annotations_topic", help="Topic name for pose annotations",
                        type=lambda value: value if value.lower() != "none" else None, 
                        default="/opendr/poses")
    parser.add_argument("-c", "--output_category_topic", help="Topic name for recognized action category",
                        type=lambda value: value if value.lower() != "none" else None, 
                        default="/opendr/skeleton_recognized_action")
    parser.add_argument("-d", "--output_category_description_topic", help="Topic name for description of the "
                                                                          "recognized action category",
                        type=lambda value: value if value.lower() != "none" else None, 
                        default="/opendr/skeleton_recognized_action_description")
```